src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-daemon-list/rgw-daemon-list.component.spec.ts

```diff
@@ -29,12 +36,42 @@ describe('RgwDaemonListComponent', () => {
   });
 
   beforeEach(() => {
+    getPermissionsSpy = spyOn(TestBed.get(AuthStorageService), 'getPermissions');
+    getPermissionsSpy.and.returnValue(new Permissions({}));
+    getRealmsSpy = spyOn(TestBed.get(RgwSiteService), 'get');
+    getRealmsSpy.and.returnValue(of([]));
     fixture = TestBed.createComponent(RgwDaemonListComponent);
     component = fixture.componentInstance;
-    fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should only show Daemons List tab', () => {
+    fixture.detectChanges();
+
+    const tabs = fixture.debugElement.nativeElement.querySelectorAll('tab');
+    expect(tabs.length).toEqual(1);
+    expect(tabs[0].getAttribute('heading')).toEqual('Daemons List');
```

Comment:
How about putting this part into a function like

```ts
  const expectTabsAndHeading = (length: number, heading: string) => {
    const tabs = fixture.debugElement.nativeElement.querySelectorAll('tab');
    expect(tabs.length).toEqual(length);
    expect(tabs[0].getAttribute('heading')).toEqual(heading);
 }
```

Reply:
Done.